community/imapsync/APKBUILD

```diff
@@ -1,22 +1,22 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=imapsync
-pkgver=1.678
-pkgrel=1
+pkgver=1.727
+pkgrel=0
 pkgdesc="IMAP synchronisation, sync, copy or migration tool"
 url="http://imapsync.lamiral.info"
 arch="noarch"
 license="custom"
 depends="perl-digest-hmac perl-authen-ntlm perl-io-compress perl-data-dumper \
 	perl-data-uniqid perl-digest-md5 perl-file-copy-recursive perl-io-socket-inet6 \
```

Comment:
`perl-io-socket-inet6` is in testing, so it must be moved to the community first.